docs/site/components/docs/docs.js

```diff
@@ -149,6 +149,8 @@ angular
                 }),
               description: $sce.trustAsHtml(
                   formatHtml(detectLinks(detectModules(obj.description.full)))),
+              lineNum: obj.codeStart,
+              lineNumLink: obj.codeStart ? '#L' + obj.codeStart : '',
```

Comment:
Super impressed you were able to work your way through this file. I hope to improve the maintainability of the site soon.